pom.xml

```diff
@@ -442,29 +441,6 @@
       <!-- =============================== -->
       <!-- Required dependencies -->
       <!-- =============================== -->
-      <dependency>
-        <groupId>commons-logging</groupId>
-        <artifactId>commons-logging</artifactId>
-        <version>${commons-logging-version}</version>
-        <exclusions>
-          <exclusion>
-            <groupId>avalon-framework</groupId>
-            <artifactId>avalon-framework</artifactId>
-          </exclusion>
-          <exclusion>
-            <groupId>logkit</groupId>
-            <artifactId>logkit</artifactId>
-          </exclusion>
-          <exclusion>
-            <groupId>log4j</groupId>
-            <artifactId>log4j</artifactId>
-          </exclusion>
-          <exclusion>
-            <groupId>javax.servlet</groupId>
-            <artifactId>servlet-api</artifactId>
-          </exclusion>
-        </exclusions>
-      </dependency>
```

Comment:
Removal of this very long standing set of (https://github.com/apache/activemq/commit/735dc7a230b15044c7fcefbe1f762b5ed13b3132) exclusions is presumably why all the log4j dependencies popped up for things referencing activeio. which uses commons-logging.

Removal of the dependencyManagement entry also explains why the instances of commons-logging are older than those present in 5.16.4

Reply:
I remember now, I removed from the dependency management because not used by ActiveMQ directly. However, indeed, it's important for dependency that use commons-logging by transitity (like activeio). I readding the commons-logging dependency management.